backend/src/main/java/com/eshka/mapper/SubjectMapper.java

```diff
@@ -3,14 +3,24 @@
 import com.eshka.dto.request.SubjectRequest;
 import com.eshka.dto.response.SubjectResponse;
 import com.eshka.entity.Subject;
+import com.eshka.service.QueueService;
+import com.eshka.service.SubjectService;
+import com.eshka.service.UserService;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Mappings;
 import org.mapstruct.factory.Mappers;
 
-@Mapper()
 public interface SubjectMapper {
     SubjectMapper INSTANCE = Mappers.getMapper(SubjectMapper.class);
 
+    @Mappings({
+            @Mapping(target = "teamId", source = "entity.team"),
+    })
     SubjectResponse subjectToSubjectResponse(Subject entity);
 
+    @Mappings({
+            @Mapping(source = "teamId", target = "team"),
```

Comment:
You forget aboout id:
`...source = "entity.team.id"),`

Reply:
fixed in the next commit